src/gui/editors/SongEditor.cpp

```diff
@@ -625,6 +626,14 @@ void SongEditor::zoomingChanged()
 
 
 
+void SongEditor::timeLineHeigthChanged()
```

Comment:
I think we could add a virtual `timeLineWidget` getter instead of this signal, member variable and slot. But that's ok too.